task3/src/scripts/users.sql

```diff
@@ -0,0 +1,68 @@
+--
```

Comment:
Is it desired SQL script from our task? (Write SQL script which will create Users table in the DB and fillit in with predefined users’collection.)

Reply:
Yes

Reply:
👍